gmmp/templates/emails/welcome_password_reset.html

```diff
@@ -0,0 +1,422 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta name="viewport" content="width=device-width" />
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+    <title>Simple Transactional Email</title>
```

Comment:
we can change this title.

Reply:
Yes... I am not fully through with the template yet as it's not being rendered well in the email. As soon as that's done, I'll fix the write up. @esirK